opencga-app/src/main/java/org/opencb/opencga/app/cli/main/options/JobsCommandOptions.java

```diff
@@ -313,8 +318,8 @@ public class SearchCommandOptions {
         @Parameter(names = {"--user-id"}, description = "User that created the job", required = false, arity = 1)
         public String userId; 
     
-        @Parameter(names = {"--priority"}, description = "Priority of the job", required = false, arity = 1)
-        public String priority; 
+        @Parameter(names = {"--job-priority"}, description = "Priority of the job", required = false, arity = 1)
```

Comment:
same here, priority instead of job-priority

Reply:
Fixed